webpack.config.js

```diff
@@ -41,10 +41,14 @@ var webpack_opts = {
 			}
 		})
 	],
+  devtool: 'source-map',
 	module: {
-		loaders: [{ test: /\.ts$/, loaders: ['babel-loader', 'awesome-typescript-loader'] }]
+		loaders: [{
```

Comment:
Can you just fixed indentation on this area.
looks very wired.

except for that it looks great, thanks!
wanted to do it for long time but didn't have the time for it..